meson.build

```diff
@@ -2895,14 +2899,37 @@ meson_install_args = meson_args + ['install'] + {
     'muon': []
 }[meson_impl]
 
+# since install_test_files are depending tmp_install
+# be sure about installing test files later
+tmp_install_priority = 100
 test('tmp_install',
     meson_bin, args: meson_install_args ,
     env: {'DESTDIR':test_install_destdir},
-    priority: 100,
+    priority: tmp_install_priority,
     timeout: 300,
     is_parallel: false,
     suite: ['setup'])
 
+# get full paths of test_install_libs to copy them
+test_install_libs_fp = []
+foreach lib: test_install_libs
+  test_install_libs_fp += lib.full_path()
+endforeach
+
+install_test_files_wrapper = files('src/test/install_test_files_wrapper')
+test('install_test_files',
+    python, args: [
+      install_test_files_wrapper,
+      '--install_path_files', '@0@/@1@/@2@'.format(
+        test_install_destdir, dir_prefix, contrib_data_args['install_dir']),
+      '--install_path_libs', '@0@/@1@/@2@'.format(
+        test_install_destdir, dir_prefix, dir_lib_pkg),
+      '--install_files', test_install_files,
+      '--install_libs', test_install_libs_fp,
+    ],
+    priority: tmp_install_priority - 1,
```

Comment:
Why is that important?

Reply:
Because some of the tests are depending these installation files, also installation paths are created at 'tmp_install'; so I wanted to run 'install_test_files' after 'tmp_install' but before running these dependent tests.